scss/_tables.scss

```diff
@@ -8,11 +8,15 @@
   color: $table-color;
   vertical-align: $table-cell-vertical-align;
   background-color: $table-bg; // Reset for nesting within parents with `background-color`.
+  border-color: $table-border-color;
 
   th,
   td {
     padding: $table-cell-padding;
-    border-bottom: $table-border-width solid $table-border-color;
+    // Table cells will only inherit if you use longhand properties
```

Comment:
Would some CSS minifiers (i.e. cssnano, et al), potentially munge these by trying to merge them into a single shorthand property?

Reply:
We use clean-css, which did in fact recombine these. I've disabled that rule for now.

Reply:
Maybe we should disable this optimization globally @mdo?

Reply:
Maybe there should be a note in the SCSS part of the docs about SASS compilation and minifiers possibly interfering with styles, as some users may be the source SCSS files and a different CSS minifier other than `clean-css` (i.e. `cssnano`, etc)

Reply:
Would you mind opening an issue for that @tmorehouse?

Reply:
OK...  will do

Reply:
Opened #30429